arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/stream/StreamTest.kt

```diff
@@ -768,15 +768,15 @@ class StreamTest : StreamSpec(spec = {
             when (uncons1) {
               null -> Pull.output1(acc)
               else -> Pull.output1(uncons1.head)
-                .flatMap { loop(acc + uncons1.head, uncons1.tail) }
+                .append { loop(acc + uncons1.head, uncons1.tail) }
```

Comment:
This doesn't change anything.
`flatMap == append` if the stream is doesn't interrupt/fail.